deploy/post_deploy_testing/cypress/support/commands.js

```diff
@@ -86,13 +85,28 @@ const auth0UserIds = {
     'u4dntestcypress@gmail.com': 'google-oauth2|104145819796576369116'
 };
 
+
+Cypress.Commands.add('signJWT', (auth0client, auth0secret, email, sub) => {
+    const secret = new TextEncoder().encode(auth0secret);
+    const jwt = new jose.SignJWT({ 'email': email, 'email_verified': true });
+    const token = jwt
+        .setProtectedHeader({ alg: 'HS256', typ: 'JWT' })
+        .setIssuedAt()
+        .setIssuer('https://hms-dbmi.auth0.com/')
```

Comment:
It would be strongly preferred to grab this info from `/auth0_config` over hard-coding, we may need to change this domain eventually.

Reply:
@willronchetti  We rewrited the code block, now it grabs the settings from /auth0_config.